docs/guide/start-installation.md

```diff
@@ -37,7 +37,7 @@ by running `composer self-update`.
 
 With Composer installed, you can install Yii by running the following commands under a Web-accessible folder:
 
-    composer global require "fxp/composer-asset-plugin:~1.0.0"
+    composer global require "fxp/composer-asset-plugin:~1.0.3"
```

Comment:
Depends on what they do with 1.1 ;-) But yeah, that also would pick up 1.0.3 as the latest.